packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -50,281 +51,288 @@ export const Default = {
 };
 
 /**
- * The sd-flipcard can be displayed in several ways using the `front-variant` and `back-variant` attributes. This example shows the usage `front-variant` attribute.
+ * Use the `variant` attribute to set the color variant for front or back.
+ * - empty (default)
+ * - primary
+ * - neutral-100
+ * - gradient-light-top
+ * - gradient-light-bottom
+ * - gradient-dark-top
+ * - gradient-dark-bottom
```

Comment:
```suggestion
 * Use the `variant` attribute to set the color variant for front or back.
 * - primary
 * - neutral-100
 * - gradient-light-top
 * - gradient-light-bottom
 * - gradient-dark-top
 * - gradient-dark-bottom
```

Reply:
@yoezlem, eventhough we don't have the empty variant for design, we do have it for code therefore I believe it is import to mention it in docs